src/NServiceBus.NHibernate.AcceptanceTests-SqlTransport/NServiceBus.NHibernate.AcceptanceTests-SqlTransport.csproj

```diff
@@ -13,8 +17,12 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
     <PackageReference Include="NServiceBus.AcceptanceTests.Sources" Version="7.0.0" />
     <PackageReference Include="NServiceBus.SqlServer" Version="4.0.0" />
-    <PackageReference Include="NUnit" Version="3.*" />
-    <PackageReference Include="NUnit3TestAdapter" Version="3.8.0-alpha1" />
+    <PackageReference Include="NServiceBus" Version="7.0.0-*" />
+    <PackageReference Include="NServiceBus.SqlServer" Version="4.0.0-beta0004" />
+    <PackageReference Include="NServiceBus.AcceptanceTesting" Version="7.0.0-*" />
+    <PackageReference Include="NServiceBus.AcceptanceTests.Sources" Version="7.0.0-*" />
```

Comment:
Shouldn't these be NServiceBus (and friends) 7.* and SqlT 4.* rather than prerelease packages?